.github/workflows/test.yml

```diff
@@ -185,3 +185,25 @@ jobs:
       - name: Test NetworkX
         run: |
           pytest -n auto --doctest-modules --durations=10 --pyargs networkx
+
+  slow-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+      - name: Set up Python
+        uses: actions/setup-python@v6
+        with:
+          python-version: "3.13"
+
+      - name: Install packages
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install .[default,test]
```

Comment:
Should this include the `extra` extra as well? None of the slow tests requires it right now, but that could change in the future and it might fall off the radar.

Reply:
It shouldn't be necessary